en/docs/develop/using-the-configuration-management-rest-apis.md

```diff
@@ -173,6 +174,10 @@ The section below describes each concept in more detail.
      An attribute is created for an already existing resource. It
     contains a key and a value.
 
+-   **File -**
```

Comment:
Since this is a new API shall we document this in the swagger based document patttern

Reply:
Configuration Management Rest API is not in the standards of IS-5.10 API's since in future will be releasing a v2 of this API with swagger based documentation